include/v8-profiler.h

```diff
@@ -175,6 +180,19 @@ class V8_EXPORT CpuProfile {
 
 
 /**
+ * HeapEventXDK contains the latest chunk of heap info
+ */
+class V8_EXPORT HeapEventXDK {
+ public:
+  const char* getSymbols();
```

Comment:
Why do you return const char pointer instead of Local<String> like it is done in CpuProfileNode?